sysml.library/Domain Libraries/Analysis/StateSpaceRepresentation.sysml

```diff
@@ -53,23 +57,25 @@ package StateSpaceRepresentation {
                 input = NextState::input,
                 initialState = NextState::stateSpace,
                 timeInterval = NextState::timeStep
-            ) return resultState = result;  
+            ) return resultState = result;
         }
-    }  
+
+        // ContinuousStateSpaceDynamics may cause zero crossing anomaly.
+        event occurrence zeroCrossingEvent[0..1] : ZeroCrossingEventDef;
```

Comment:
I intend to specify that every time ContinuousStateSpaceDynamics is executed, it may or may not occur zeroCrossingEvent.  Does this statement align to it?

Reply:
Yes, this will work, as long as your intent is to record at most 1 zero crossing event for each performance.

Reply:
I thought for each performance of `ContinuousStateSpaceDynamics`, `zeroCrossingEvent` may occur at most once.  But that is not correct because we can do `getOutput()` as well and it must not cause `zeroCrossingEvent`.  So I will move `zeroCrossingEvent` into `nextState`

Reply:
`zeroCrossingEvent` occurs at most once for each performance of `nextState` but it is not true.  In the typical implementation, so-called zero-crossing vector is computed at the beginning and in the end of each tilmestep to detect and estimate the point or zero-crossing.  So `[0..*]` is more general to capture that as you suggested.  I'll fix it.